src/SmartComponents/Inventory/applications/advisor/Advisor.js

```diff
@@ -74,7 +74,7 @@ class InventoryRuleList extends Component {
     }
 
     sortActiveReports (activeReports) {
-        const reports = activeReports;
+        const reports = Object.assign({}, activeReports);
```

Comment:
Isn't spread operator better here? `{...activeReports}`

Reply:
OOOOO seee i had this same conversation with @dkuc https://stackoverflow.com/questions/32925460/object-spread-vs-object-assign had some thoughts...  didn't ultimately come to any conclusion...